wpiutil/src/main/native/include/wpi/ct_string.h

```diff
@@ -63,6 +63,34 @@ struct ct_string {
     }
   }
 
+  constexpr bool operator==(const ct_string<Char, Traits, N>&) const = default;
+
+  constexpr bool operator==(std::basic_string_view<Char, Traits> rhs) const {
```

Comment:
Do we need to add all the other string comparisons, e.g. `std::string<Char, Traits>` and `const Char*`?